NEWS.md

```diff
@@ -3,6 +3,211 @@ XGBoost Change Log
 
 This file records the changes in xgboost library in reverse chronological order.
 
+## v1.4.0 (2021.04.12)
+
```

Comment:
```suggestion
### Introduction of pre-built binary package for R, with GPU support
Starting with release 1.4.0, users now have the option of installing `{xgboost}` without having to build it from the source. This is particularly advantageous for users who want to take advantage of the GPU algorithm (`gpu_hist`), as previously they'd have to build `{xgboost}` from the source using CMake and NVCC. Now installing `{xgboost}` with GPU support is as easy as:
`R CMD INSTALL ./xgboost_r_gpu_linux.tar.gz`

See the instructions at https://xgboost.readthedocs.io/en/latest/build.html
```

Reply:
@hcho3 Thanks for reminding me, why use `{xgboost}`, specifically the `{}`?

Reply:
This is the convention used in the R user community to indicate the name of R packages. Example: `{caret}`

Reply:
Got it, thanks for the explanation.